main.go

```diff
@@ -80,13 +82,15 @@ func run() error {
 	if err != nil {
 		return fmt.Errorf("unable to create client: %v", err)
 	}
-	imageClient, err := imageclientset.NewForConfig(config)
+	// Config and Client to access release images
+	releaseConfig, err := loadKubeconfigFromFlagOrDefault(opt.ReleaseClusterKubeconfig, config)
```

Comment:
you need to default this value to BuildClusterKubeconfig if empty 

```
releaseKubeconfig := opt.ReleaseClusterKubeconfig
if len(releaseKubeconfig) == 0  {
  releaseKubeconfig = opt.BuildClusterKubeconfig
}
```

so that this has back compat

Reply:
Oh, nm.  Got it.